package.json

```diff
@@ -12,47 +14,54 @@
     "LICENSE.txt"
   ],
   "scripts": {
-    "prepare": "run-s clean build",
-    "prepublishOnly": "npm test",
-    "preversion": "yarn build:prod",
-    "postversion": "git push && git push --tags && npm publish",
-    "test": "eslint --ext .ts 'src/**/*.ts'",
     "assets": "run-s clean build assets:version assets:zip",
     "assets:version": "node scripts/assets-append-version.js",
-    "assets:zip": "zip -x '*types*' -j -r $(cat package.json | npx json name)-$(cat package.json | npx json version).zip dist",
-    "clean": "rm -rf dist *.tgz *.zip",
+    "assets:zip": "zip -x '*types*' -j -r $(node scripts/dist-name.js)-$(cat package.json | npx json version).zip dist",
     "build": "run-p build:*",
     "build:dev": "rollup --config rollup.config.js",
+    "build:dts": "tsc --project src/tsconfig-dts.json",
     "build:prod": "rollup --config rollup.config.js --environment BUILD:production",
+    "clean": "rimraf dist *.tgz *.zip",
     "format": "run-p format:*",
-    "format:scss": "prettier --parser scss --write 'src/sass/**/*.scss'",
-    "format:ts": "eslint --ext .ts --fix 'src/**/*.ts'",
-    "watch": "npm run build:dev && run-p watch:*",
-    "watch:sass": "onchange 'src/sass/**/*.scss' -- npm run build:dev",
-    "watch:ts": "onchange 'src/**/*.ts' -- rollup --config rollup.config.js"
+    "format:scss": "prettier --parser scss --write \"src/sass/**/*.scss\"",
+    "format:ts": "eslint --ext .ts --fix \"src/**/*.ts\"",
+    "postversion": "git push --tags",
+    "prepare": "run-s clean build",
+    "prepublishOnly": "npm test",
+    "server": "http-server -c-1 -o /test/browser.html",
+    "start": "run-p watch server",
+    "test": "eslint --ext .ts \"src/**/*.ts\"",
+    "watch": "run-p watch:*",
+    "watch:sass": "onchange --initial --kill \"src/sass/**/*.scss\" -- npm run build:dev",
+    "watch:ts": "onchange --initial --kill \"src/**/*.ts\" -- rollup --config rollup.config.js"
```

Comment:
Is this just a sort, or is there anything to review here?

Reply:
Some changes to match the latest v4 plugin template:
https://github.com/tweakpane/plugin-template/blob/main/package.json